src/Migration.php

```diff
@@ -1561,4 +1561,35 @@ private function migrateSearchOptions()
             }
         }
     }
+
+    /**
+     * Helper to create a simple link table between two itemtypes
+     * The table will contain 3 columns :
+     * - id (primary key)
+     * - foreign key 1 (first itemtype)
+     * - foreign key 2 (second itemtype)
+     *
+     * @param string $table Table name
+     * @param string $class_1 First itemtype (CommonDBTM)
+     * @param string $class_2 Second itemtype (CommonDBTM)
+     */
+    public function createLinkTable(
+        string $table,
+        string $class_1,
+        string $class_2
+    ) {
+        $fk_1 = $class_1::getForeignKeyField();
+        $fk_2 = $class_2::getForeignKeyField();
+
+        $this->addPreQuery("
+            CREATE TABLE `$table` (
+                `id` int NOT NULL AUTO_INCREMENT,
+                `$fk_1` int NOT NULL DEFAULT '0',
+                `$fk_2` int NOT NULL DEFAULT '0',
+                PRIMARY KEY (`id`),
+                KEY `$fk_1` (`$fk_1`),
+                KEY `$fk_2` (`$fk_2`)
+            ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
```

Comment:
Should use `DBConnection::getDefaultCharset()` and `DBConnection::getDefaultCollation()` values.